philosophy.md

```diff
@@ -1,12 +1,36 @@
 # Philosophy
 
-This page contains a list of guidelines that should inform decisions related to scope, command organization and behaviour:
+WP-CLI is the command-line interface for WordPress. It provides commands for actions that can be performed in the WordPress admin (e.g. `wp theme activate` and `wp user create`), as well as commands for actions that don't have equivalent web UI (e.g. `wp cron event run`, `wp search-replace`, and `wp scaffold child-theme`).
```

Comment:
... in the WordPress admin _backend_ ?

Reply:
... that don't have _an_ equivalent web UI ?

Reply:
2942009